Makefile

```diff
@@ -52,6 +52,13 @@ format:
 	docker-compose exec web black .
 
 format-check:
+	docker-compose exec web isort -rc -tc --atomic --diff .
```

Comment:
This is a new one for me, thanks!

Reply:
@elmq0022 has a current PR https://github.com/chicagopython/chipy.org/pull/285
that adds `docker-compose exec web isort -rc -tc --atomic .` to the Makefile 'format'. So I decided to add the accompanying diff command so people can preview.